content/html/concepts/attributes/attributes.md

```diff
@@ -30,29 +30,27 @@ So for example, the `<img>` image element has a mandatory `src` attribute that c
 - `src` is the attribute name.
 - `codecademy_logo.png` is the attribute value.
 
-All HTML elements can have attributes. Multiple of them, too!
+All HTML elements can have attributes; many have more than one.
 
 ```html
 <img src="nyan_cat.gif" width="500" height="600" />
 ```
 
-**Note:** Attribute names are case insensitive, but it is a good practice to keep HTML markup lowercase.
+**Note:** Attribute names are not case-sensitive, but it is good practise to write HTML markup in lowercase.
 
 ## Unique ID Attributes
 
-In HTML, specific and unique `id` attributes can be assigned to different elements in order to differentiate between them.
+Specific and unique `id` attributes can be assigned to elements to differentiate them, and `id` values can be referenced in CSS and JavaScript to manipulate, format, and perform specific instructions on a single element alone, without affecting others of the same type.
 
-When needed, the `id` value can be called upon by CSS and JavaScript to manipulate, format, and perform specific instructions on that element and that element only.
-
-Valid `id` attributes should begin with a letter and should only contain letters (`a`-`Z`), digits (`0`-`9`), hyphens (`-`), underscores (`_`), and periods (`.`).
+Valid `id` attribute values begin with a letter and must comprise only letters (`a`-`Z`), digits (`0`-`9`), hyphens (`-`), underscores (`_`), and periods (`.`).
 
 ```html
 <h1 id="th1s_is.v4lid">Hello World</h1>
 ```
 
```

Comment:
Would be helpful to add a note here about what is convention.

Reply:
Sorry, could you expand what you mean by that? I'm not sure I understand.

Reply:
Yup! I'm speaking about id naming conventions. What do you think about adding a note that while an id like "th1s_is_v4lid" is valid, it's best to stick to one style? ..camelCase, dash-case, snake_case, and so on.

This is definitely ❓ optional

Reply:
Added at https://github.com/hughlilly/docs/commit/6e0f64a85ce8792bac8e9da6d578f48a336661a0